articles/virtual-machines/dedicated-hosts-how-to.md

```diff
@@ -176,6 +210,9 @@ $dHost = New-AzHost `
 
 Now create a VM on the host.
 
+If you would like to create a VM with Ultra disks support, make sure the host group in which the VM would be placed is Ultra SSD Enabled (Preview), then create the VM in the same host group. See [Deploy an ultra disk](disks-enable-ultra-ssd.md#deploy-an-ultra-disk) for the steps to attach an ultra disk to a VM.
```

Comment:
Lower case ultra.

I would consider changing the wording to "... make sure the host group the VM will be placed into is ultra ssd enabled (Preview). Once you've confirmed, create the VM in the same host group."

Reply:
left teh acronym capitalized, but fixed the rest as recommended.